cmake/BLTMacros.cmake

```diff
@@ -47,11 +47,23 @@ macro(blt_list_append)
          message(FATAL_ERROR "blt_list_append() requires ELEMENTS to be specified" )
     endif()
 
+    # determine if we should add the elements to the list
+    set(_shouldAdd FALSE )
+    set(_listVar "${ARGN}")         # convert macro arguments to list variable
```

Comment:
Macro arguments are not really a list.  See: https://cmake.org/cmake/help/v3.14/command/macro.html#argument-caveats

Reply:
Good catch! Thanks!